react/src/components/atoms/icons/SvgQuestionMark/index.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
```

Comment:
can you please update the icon in tooltip to use the new Icon component that just got merged in?